trunk/SUAVE/Methods/Aerodynamics/AVL/Data/Wing.py

```diff
@@ -79,11 +80,11 @@ def append_control_surface(self,control):
 class Control_Surface(Data):
 	def __defaults__(self):
 		self.tag            = 'control_surface'
-		self.gain           = 1.0
+		self.gain           = 0.0
 		self.x_hinge        = 0.0
-		self.hinge_vector   = [0.,0.,0.]
-		self.sign_duplicate = 1.0	# sign_duplicate: 1.0 or -1.0 - the sign of
-									# the duplicate control on the mirror wing.
-									# Use 1.0 for a mirrored control surface,
-									# like an elevator. Use -1.0 for an aileron.
+		self.hinge_vector   = '0. 0. 0.'
```

Comment:
No changes were made to this part of the script. I chose to leave it until I got the green light on doing control surfaces in AVL. Some of the code has already been written (and commented out scripts) however I could not find information on flap angles in the rest of the SUAVE code